### ISSUE TITLE
All the Italics are not being exported correctly

### ISSUE THREAD

**Issue by felipesanches** [MonicaRizzolli/Tomorrow#11]
https://travis-ci.com/felipesanches/Tomorrow/builds/128414009

![Screenshot at 2019-09-19 17:31:15](https://user-images.githubusercontent.com/213676/65278862-82b55580-db1c-11e9-9310-2b25ff3abb11.png)

It seems that fontmake is not understanding that these are Italics and it is simply overwriting the corresponding roman TTFs.

**Comment by felipesanches** [comment]
I tried setting style linking for the Black Italic:
<img width="598" alt="Captura de Tela 2019-09-19 às 17 37 00" src="https://user-images.githubusercontent.com/213676/65279638-2266e980-db05-11e9-99e8-68d2380d4bf5.png">

But still did not get it to export correctly. A weird result is that now the Travis build talks about the BlackItalic glyphs file being apparently exported to an "UltraBold" TTF!

We do not have an UltraBold glyphs file. So I used grep to see what's going on...

<img width="492" alt="Captura de Tela 2019-09-19 às 17 44 09" src="https://user-images.githubusercontent.com/213676/65279792-63f79480-db05-11e9-98c8-c0622f0d2368.png">

Gotcha!
